src/cli/cli.cpp

```diff
@@ -1961,6 +1961,7 @@ void Interpreter::ProcessRouterSelectionJitter(int argc, char *argv[])
     else
     {
         SuccessOrExit(error = ParseLong(argv[0], value));
+        VerifyOrExit(0 < value && value < 256, error = kThreadError_InvalidArgs);
```

Comment:
Spec defines ROUTER_SELECTION_JITTER is a random from 0 to 120s (if CLI does not configure this value, the default jitter is 120s), do we need to limit the max arg value passed via CLI is less or equal to 120 instead of 256?